plugins/htpasswd/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "useTabs": false,
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "parser": "flow",
+  "semi": true
+}
```

Comment:
Well, in next steps we should unify linting of all subproject (maybe when we import the rest), but it requires some discuss before, so this could stay as it is now

Reply:
yes, I think some dot files are outdated, we can unify in a further step.